recon/src/recon/tests.rs

```diff
@@ -1514,80 +1501,6 @@ async fn two_in_sync() {
     .await
 }
 
-#[test(tokio::test)]
-async fn dog_missing_sync() {
```

Comment:
is there a reason these tests were deleted?

Reply:
Yes, they tested if synced work when a peer knew about a key but not a value. Since that is not longer a valid state the tests are not needed.